snorkel/analysis/metrics.py

```diff
@@ -91,14 +91,33 @@ def _roc_auc_score(golds: np.ndarray, probs: np.ndarray) -> float:
     return skmetrics.roc_auc_score(golds, probs[:, 0])
 
 
+def _f1_score(golds: np.ndarray, preds: np.ndarray) -> float:
+    if golds.max() <= 1:
+        return skmetrics.f1_score(golds, preds)
```

Comment:
does this fail silently if theres multiple classes?

Reply:
nope! raises ValueError